controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1307,6 +1321,52 @@ int cli_bml::get_dcs_scan_results_caller(int numOfArgs)
     return -1;
 }
 
+/**
+ * caller function for client_get_client_list_caller
+ *
+ * @param [in] numOfArgs Num of received arguments
+ *
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list_caller(int numOfArgs)
+{
+    if (numOfArgs == 0) {
+        return client_get_client_list();
+    }
+    return -1;
+}
+
+/**
+ * caller function for client_set_client_caller
+ *
+ * @param [in] numOfArgs Num of received arguments
```

Comment:
Usually, Doxygen documentation should not have word shortcuts like "Num". Just write "Number".

Reply:
Fixed